dat/board.lua

```diff
@@ -454,6 +454,36 @@ You will still have to escort the ship and land with it to perform the repairs a
       end )
 end
 
+local function board_fcthit_apply ( eat )
+   -- Piss off nearby ships
+   board_plt:distress( player.pilot() )
+
+   local fct = board_plt:faction()
+   local loss = math.floor(-fct:hit( -board_fcthit, system.cur(), "board" ))
+   local msg
+
+   if eat==0 then
+      board_fcthit = 0
+      if loss == 0 then
```

Comment:
Probably do `loss <= 0` in case something goes wrong and it creates a negative loss.